lib/Command/Configure.php

```diff
@@ -225,7 +225,7 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 
 			case 'spryng':
 				$config->setProvider($provider);
-				/** @var ClockworkSMSConfig $providerConfig */
+				/** @var SpryngSMSConfig $providerConfig */
```

Comment:
the class is not imported. see `use` statements at the top of the file

Reply:
Oh I see, 
`use OCA\TwoFactorGateway\Service\Gateway\SMS\Provider\SpryngConfig` should be 
`use OCA\TwoFactorGateway\Service\Gateway\SMS\Provider\SpryngSMSConfig`
on top of the file